compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TaintAnalyzer.java

```diff
@@ -1146,6 +1146,10 @@ private boolean isExternalLangLibFunction(BLangInvocation invocationExpr) {
         return taintTable;
     }
 
+    private int receiverIfAttachedFunction(BLangInvocation invocationExpr) {
+        return isTaintAnalyzableAttachedFunction(invocationExpr) ? 1 : 0;
```

Comment:
Do we need a separate method for this? It is only used once.

Reply:
This is just for readablity